### ISSUE TITLE
Utils is not defined

### ISSUE THREAD

**Issue by RobAley** [grimmdude/MidiWriterJS#67]
Using the npm package, trying to send a controller change message I get :

```
this.data = Utils.numberToVariableLength(0x00).concat(Constants.CONTROLLER_CHANGE_STATUS, fields.controllerNumber, fields.controllerValue);
              ^

ReferenceError: Utils is not defined
    at new ControllerChangeEvent (/node_modules/midi-writer-js/build/index.js:89:15)
```

Changing Utils to Utils$1 works, I assume this is a build issue?

Thanks for a great lib.

**Comment by PedroAlvesV** [comment]
Just out of curiosity: why would you use such low-level commands?

**Comment by grimmdude** [comment]
Hi @RobAley,

Thanks for your message.  Interesting, could you post the full code you're using which results in this error please?

-Garrett

**Comment by grimmdude** [comment]
Hi @RobAley,

I think I see what's happening.  I just made a change and published new version `1.7.2`.  You want to try again with that new version?

-Garrett

**Comment by RobAley** [comment]
@grimmdude,

Thanks, that works great! A sample of the code I was using below for reference, I changed it to use the helper you added. No errors, and if I play the test.midi it creates with fluidsynth and a sf2 like General User I get a single snare hit as expected

@PedroAlvesV,

They're not particularly "low-level" codes, they're just less common ones more often used when working with physical midi instruments rather than midi files. My particular use is for changing instrument banks. Many sound fonts (and indeed many physical midi keyboards etc.) have more than the midi standard 128 instruments, and so they arrange them into banks of instruments. Control Code 00 ( for MSB ("most significant bit", which is all most fonts/instruments use) and 20 (for LSB, for those that use more) changes the bank, and then you select a particular instrument from within that bank with a "program change" like normal.

```
var MidiWriter = require('midi-writer-js');
var fs = require('fs');

var track = new MidiWriter.Track();

// change to instrument bank 120 (percussion)

// old code I was using :
//
// track.addEvent(new MidiWriter.ControllerChangeEvent(
//     {controllerNumber : 0, controllerValue : 120}
//   )
// );

// new helper you added :
track.controllerChange(0, 120);

// select program 24 (Electric Drums), without the bank change
// it would be Nylon Guitar in bank 0 (where the 128 midi standard
// instruments are

track.addEvent(new MidiWriter.ProgramChangeEvent({instrument : 24}));

var note = new MidiWriter.NoteEvent(
  // add single snare drum (D2) hit
  {pitch: 'D2', channel : 1, duration: '1', velocity:100}
);

track.addEvent(note);

var write = new MidiWriter.Writer(track);

fs.writeFileSync('./test.midi', write.buildFile() )

```
